src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -1111,30 +1112,21 @@ private void BuildErrorBubble()
         /// </summary>
         /// <param name="style"></param>
         /// <returns></returns>
-        internal SolidColorBrush GetWarningColor(InfoBubbleViewModel.Style style)
+        internal SolidColorBrush GetWarningColor()
         {
-            switch (style)
+            if (nodeLogic.IsInErrorState)
             {
-                case InfoBubbleViewModel.Style.None:
-                    if (IsVisible == false)
-                    {
-                        return (SolidColorBrush)SharedDictionaryManager.DynamoColorsAndBrushesDictionary["NodePreviewColor"];
-                    }
-                    break;
-                case InfoBubbleViewModel.Style.Warning:
-                case InfoBubbleViewModel.Style.WarningCondensed:
-                    return (SolidColorBrush)SharedDictionaryManager.DynamoColorsAndBrushesDictionary["NodeWarningColor"];
-                case InfoBubbleViewModel.Style.Error:
-                case InfoBubbleViewModel.Style.ErrorCondensed:
-                    return (SolidColorBrush)SharedDictionaryManager.DynamoColorsAndBrushesDictionary["NodeErrorColor"];
-                case InfoBubbleViewModel.Style.Info:
-                    return (SolidColorBrush)SharedDictionaryManager.DynamoColorsAndBrushesDictionary["NodeInfoColor"];
```

Comment:
@saintentropy - after your changes I do not see this color set anywhere.

Reply:
`NodeInfoColor`

Reply:
@Amoursol confirms that this color was planned but the functionality for setting it is not exposed, so I think it's okay to remove it, we'll have to refactor this method or add an info state to the node later - or something like that.

Reply:
Correct.  I took it out as it wasn't included yet.  We will do as you say when the state is added.

Reply:
Agree on above, let's leave it out for now